### PR TITLE
Changed the script so the user can generate a map with date range.

### PR DESCRIPTION
**Pull request by almedin79**
I am currently learning github and python if someone has any tips how to make the code cleaner would be nice. :)

### CODE REVIEW

**Comment by luka1199** [comment]
I don't know if this is that usefull since with your implementation you can only pick one day.

**Comment by arjvik** [comment]
Just an FYI, this is not ready to merge in any way. Nonetheless, good idea!

Some quick tips:
- Fix the README by removing all extraneous comments you made. These shouldn't appear in the final README of this script, so you shouldn't make those changes in your PR
- Make the example more descriptive
- Remove all your extraneous comments in the code
- Better more descriptive variable names
- Make the date range optional, perhaps with a flag

**Comment by almedin79** [comment]
Mixed up the word and sketch and draft for some reason. The code works even if its messy. The location html file is easy to merge with someone else so i tough that i can merge my friends trip to Denmark we did together for memory instead of pictures. I will try to fix all that code on the weekend. 
Any tips on how to discuss code before a pull request ,

**Comment by luka1199** [comment]
To discuss the code here is fine. It would probably be best to add argparse (#15) for your feature so that it is easier to add features in the future (e.g. #12).

File: README.md [hunk 1]
```diff
@@ -29,9 +29,14 @@ pip install -r requirements.txt
 ### 5. Run the Script
 In the same command prompt or Terminal window, type the following, and press enter:
 ```
-python geo_heatmap.py <file>
+python geo_heatmap.py <file> yyyy month day yyyy month day
```
Comment (dhimmel) [inline README.md]: I think it'd make more sense to make two optional named arguments like:

```
--min-date=yyyy-mm-dd
--max-date=yyyy-mm-dd
```

iso date format with hyphens

**Comment by luka1199** [comment]
I'm going to close this since I already implemented it.

**Comment by almedin79** [comment]
Sorry had lot last week  i have just started to improve the code right now.

**Pull request closed without merging**